pol-core/support/scripts/uo.em

```diff
@@ -395,7 +395,7 @@ SetRegionLightLevel( regionname, lightlevel );
 SetRegionWeatherLevel( region, type, severity, aux := 0, lightoverride := -1);
 SetScriptController( who );
 SingleClick( who, what );
-Shutdown();
+Shutdown( exitCode := 0 );
```

Comment:
Strongly recommend increasing the .ecl version.  If any script that calls Shutdown isn't recompiled, the script will be killed rather than calling the Shutdown method.

Reply:
I thought https://github.com/polserver/polserver/pull/176/files#diff-be892f55676b7e6d68bfc47ff7710570R4787 would cover optional arguments?

Reply:
It turns out it won't, because it won't get that far.

Here's what would happen:
1. Previously-compiled script passes 0 arguments to Shutdown
2. `Executor::ins_func` [calls getParams](https://github.com/polserver/polserver/blob/master/pol-core/bscript/executor.cpp#L2272) with the number of parameters for the module function (1)
3. `Executor::getParams` [sees an empty value stack](https://github.com/polserver/polserver/blob/master/pol-core/bscript/executor.cpp#L181) and kills the script.

Reply:
Addressed by increasing `ESCRIPT_FILE_VER_CURRENT` to `0x10`